src/index.html

```diff
@@ -11,7 +11,174 @@
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="field">
+          <span>Surname:</span>
+          <input
+          type="text"
+          name="surname"
+          required
+          autocomplete="off"
+          >
+        </label>
+
+        <label  class="field">
+          <span>Name:</span>
+          <input
+          type="text"
+          name="name"
+          required
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="field">
+          <span>How old are You?</span>
+          <input
+          type="number"
+          name="Age"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+
+        <label class="field">
+          <span>Full date of birth:</span>
+          <input
+          type="date"
+          name="dateOfBirth"
+          required
+          >
+        </label>
+
+        <label class="field">
+          <span>I accept the term of the agreement</span>
+          <input
+          type="checkbox"
+          name="terms"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="field">
+          <span>E-mail:</span>
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="field">
+          <span>Password:</span>
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="15"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+
+        <legend>An interesting fact about you!</legend>
+        <div class="field">
+          <span>Do you love cats?</span>
+          <label>
+            <input
+            type="radio"
+            name="cat"
+            value="yes"
+            required
+            >
+            <span>Yes</span>
+          </label>
+
+          <label>
+            <input
+            type="radio"
+            name="cat"
+            value="no"
+            required
+            >
+            <span>No</span>
+          </label>
+        </div>
+
+        <label class="field">
+          <span> What is your favorite color? </span>
+          <input type="color" name="color">
+        </label>
+
+
+        <label class="field">
+          <span>What time do you go to bed?</span>
+          <input type="time" name="bedTime">
+        </label>
+
+
+        <label class="field">
+          <span>What are your favorite brand of cars?</span>
+          <select
+            name="carBrand"
+            required
+            multiple
+            >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
```

Comment:
indents and
better pass lower-cased `value` - it'll be easier to process on server-side (no need for letter-case conversion)
```suggestion
          <select
            name="carBrand"
            required
            multiple
          >
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
          </select>
```